javascript/ql/src/Declarations/SuspiciousMethodNameDeclaration.qhelp

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE qhelp PUBLIC
+  "-//Semmle//qhelp//EN"
+  "qhelp.dtd">
+<qhelp>
+<overview>
+<p>
+In TypeScript the keywords <code>constructor</code> and <code>new</code> are
+used to declare constructors in classes and interfaces respectively. 
+However, by using the wrong keyword a programmer can accidentally declare e.g. 
+a method called <code>constructor</code> inside an interface.
```

Comment:
```
However, by using the wrong keyword a programmer can accidentally declare e.g. 
a method called <code>constructor</code> inside an interface.
```

There's only two cases here, I think it would be clearer to spell them out:

```
However, a member declaration with the name <code>new</code> in an interface or <code>constructor</code> in a class,
will declare ordinary methods named <code>new</code> or <code>constructor</code> rather 
than constructors.
```